packages/@aws-cdk/toolkit-lib/test/api/hotswap/ecs-services-hotswap-deployments.test.ts

```diff
@@ -724,5 +724,96 @@ describe.each([
       },
       forceNewDeployment: true,
     });
+    expect(mockECSClient).toHaveReceivedCommandWith(DescribeServicesCommand, {
+      cluster: 'arn:aws:ecs:region:account:service/my-cluster',
+      services: ['arn:aws:ecs:region:account:service/my-cluster/my-service'],
+    });
```

Comment:
I just noticed this assertion was missing, to make sure we actually call the waiter.